tests/test_allow_unmocked_https.js

```diff
@@ -55,7 +55,6 @@ test('Nock with allowUnmocked, url match and query false', async t => {
 
   nock(`${url}`, { allowUnmocked: true })
     .get('/')
-    .query(false)
```

Comment:
Ah, we're removing support for this undocumented feature, correct? Could you update the test title to remove "query false"?

Reply:
This test just seems to be a duplicate of the test above so I'm going to remove it completely.